engine/runtime/src/test/java/org/enso/interpreter/test/MetaObjectTest.java

```diff
@@ -77,8 +77,10 @@ public void checkAllConstantGenValuesArePresent() throws Exception {
     var g = ValuesGenerator.create(ctx);
     var expecting = new HashSet<String>();
     for (var f : ConstantsGen.class.getFields()) {
-      var s = (String) f.get(null);
-      expecting.add(s);
+      if (!f.getName().endsWith("_BUILTIN")) {
```

Comment:
What is this?

Reply:
This is so that we don't have to write (by hand) FQN of builtin types. We generate `ConstantsGen` for types from stdlib but if one does not import stdlib builtin types are essentially `Standard.Builtin.Main.XYZ`. So every type which is in stdlib, say `Standard.Base.Foo.Bar.XYZ` has an equivalent in `Standard.Builtin.Main.XYZ`. Rather than trying to keep those up-to-date, processor adds those to `ConstantsGen`.

You might ask why do we need this at all?
Well, in tests (but also in a real application), if you don't import stdlib you operate only on builtins and the reported type shouldn't really be some type from stdlib, should it? That is why the type reported by `TypeOfNode` is an actual type, not some hardcoded string (like `Types.getName`). Hence the changes in tests (the changed ones don't import stdlib).